### ISSUE TITLE
[BUG] Dialyzer error for my app, using PromEx

### ISSUE THREAD

**Issue by djthread** [akoutmos/prom_ex#24]
**Describe the bug**
My `use PromEx, otp_app: :my_app` is causing the following dialyzer error:

```
lib/drug_mule/prom_ex.ex:50:no_return
Function init/1 has no local return.
```

**To Reproduce**

1.  Create a new phoenix app
2. Follow PromEx setup instructions

**Expected behavior**
no dialyzer warnings

**Environment**

- Elixir version: 1.11.2
- Erlang/OTP version:
- Grafana version:
- Prometheus version:

**Additional context**

**Comment by stefanluptak** [comment]
Same here.

```
Erlang/OTP 23 [erts-11.1.6] [source] [64-bit] [smp:8:8] [ds:8:8:10] [async-threads:1]
Elixir 1.11.3 (compiled with Erlang/OTP 23)
```

**Comment by akoutmos** [comment]
Thanks for opening up the issue @djthread and thanks for verifying @stefanluptak ! I was able to root cause the issue and will have a fix out along with a new version in the coming days :).

**Comment by akoutmos** [comment]
Just published version 0.1.14-beta to Hex that includes the fix for this as well as a LiveView plugin and Grafana dashboard. Feel free to reopen the issue if the dialyzer error does not go away :).
